package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "aws-lambda-ric",
   "version": "2.0.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
   "requires": true,
   "packages": {
     "": {
       "name": "aws-lambda-ric",
-      "version": "1.1.0",
+      "version": "2.0.0",
```

Comment:
Is this an expected change?

Reply:
Yes, 2.0.0 is the current version of this library.